azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -818,10 +838,8 @@ public static void main(String[] args) throws Exception {
     loadViewerPlugins(root, viewerPluginDir, app.getVelocityEngine());
 
     // triggerplugin
-    String triggerPluginDir =
-        azkabanSettings.getString("trigger.plugin.dir", "plugins/triggers");
-    Map<String, TriggerPlugin> triggerPlugins =
-        loadTriggerPlugins(root, triggerPluginDir, app);
+    String triggerPluginDir = azkabanSettings.getString("trigger.plugin.dir", "plugins/triggers");
```

Comment:
I guess this plugin was proposed by Evan before, and we may retire these code in future, right?